sdk/servicebus/azure-servicebus/azure/servicebus/_transport/_base.py

```diff
@@ -225,6 +227,35 @@ def iter_contextual_wrapper(
         :param int or None max_wait_time: The maximum wait time in seconds for the batch to be received.
         """
 
+    @staticmethod
+    @abstractmethod
+    def turn_on_prefetch(
+        receiver: "ServiceBusReceiver"
+    ) -> Optional[Literal[1]]:
+        """
+        Not used for uamqp. To be used with `__next__` calls.
```

Comment:
since we dont want this in uamqp, can we just move the implementations in to `pyamqp_transport` and then you know it exists there only ?

Reply:
nvm, I see now it would end up affecting uamqp as the check is done on the receiver level

Reply:
will update docstring to explain why it's needed!

Reply:
no longer needed, moving prefetch+1 logic into _pyamqp